test_scripts/Defects/5_0/1178_Core_sends_an_onAppInterfaceUnregistered_notification_when_UnregisterAppInterface_is_sent_from_Mobile.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- User story: https://github.com/SmartDeviceLink/sdl_core/issues/1178
+--
+-- Description:
+-- Core sends an onAppInterfaceUnregistered notification when UnregisterAppInterface is sent from Mobile
+--
+-- It seems as though we have an issue with core sending both an onAppInterfaceUnregistered 
+-- notification and an UnregisterAppInterface response when the mobile proxy sends over 
+-- UnregisterAppInterface. This should not be the case, as the notification should only be sent 
+-- if there is an unregistration from the core side that was not caused by the mobile side. 
+-- The notification also has a reason of -1, which is not a valid value.
+-- We should remove the notification from being sent when the UnregisterAppInterface RPC is sent.
```

Comment:
@KrupenichOleg  Please remove lines 7 trough 12 since preconditions, steps and expected result will be enough

Reply:
@dboltovskyi all done in commit [3a40003](https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2109/commits/3a40003fb6428e28fa8271b95fa21b83bb22e43a)